app/components/post_list/date_separator/index.tsx

```diff
@@ -20,11 +20,7 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
         container: {
             alignItems: 'center',
             flexDirection: 'row',
-            height: 28,
-            marginTop: 8,
-        },
-        dateContainer: {
-            marginHorizontal: 15,
```

Comment:
why this change and how does it affect the current post list?

Reply:
The gap between components should be 16pt, according to UX review, it shouldn't have any other effect different than that in the post list. But we'll have to revisit when one of the 2 PRs (this, or channel intro) is merged.